packages/picasso/src/List/List.tsx

```diff
@@ -23,9 +27,27 @@ const Tags = {
   ordered: 'ol',
 } as const
 
+const getDefaultType = (variant: Props['variant'], level: number) => {
+  if (variant === 'unordered') {
+    const isOddLevel = level % 2 === 0
+
+    return isOddLevel ? 'disc' : 'circle'
```

Comment:
btw, we can improve it like this: 
```suggestion
    return level % 2 ?  'circle' : 'disc'
```

Reply:
The first version I did was exactly like this, but I thought a bit about it, and putting on a variable makes the code more readable and self-documented. I prefer having the temporary variable, wdyt?

Reply:
no objections against a temp variable :)